README.md

```diff
@@ -116,103 +101,89 @@ Create a file in `utils/` called `jadenCase.test.js`, then write two tests:
 
 ### Part Two: Testing React components
 
-So how do we test React components? We're going to try and test them as closely to how they'll really be used as possible. That means rendering them in a "DOM" and simulating events to test interaction.
+So how do we test React components? We're going to try and test them as closely to how they'll really be used as possible. That means rendering them in a "DOM" and firing events to test user interaction.
 
 #### jsdom
 
 Jest comes with [jsdom](https://github.com/jsdom/jsdom) set-up. This is a Node implementation of most of the browser APIs needed to interact with the DOM. This means we can treat our Jest environment as if it were a browser.
 
+#### React Testing Library
+
+We'll be using [React Testing Library](https://testing-library.com/docs/react-testing-library/intro) (RTL) to render our React components to jsdom, fire events and find things on the "page".
+
 #### Rendering
 
-We use `ReactDOM.render` to render our component into a node in our "DOM" (we'll have to create this node via JS since we don't have an actual HTML DOM):
+We can render our components to jsdom using RTL's [`render`](https://testing-library.com/docs/react-testing-library/api#render) method.
```

Comment:
Maybe say React Testing Library instead of RTL here (or mention earlier that RTL = React Testing Library)

Reply:
I did on line 112 :)